shipyard-controller/handler/projecthandler.go

```diff
@@ -100,11 +101,11 @@ func (ph *ProjectHandler) GetProjectByName(c *gin.Context) {
 	project, err := ph.ProjectManager.GetByName(projectName)
 	if err != nil {
 		if project == nil && err == ErrProjectNotFound {
```

Comment:
Should we change also this line to use `errors.Is()`? Like here: https://github.com/keptn/keptn/pull/7080/files#diff-6d643b14d1fbb5301d0aed6dd99b501b9e249daa4dceb265749974aeb853dd24R201

Reply:
sure